### PR TITLE
Reference app.js and app.css in layout

### PR DESCRIPTION
**Pull request by NduatiK**
- Modify backoffice.html.eex to reference app.css and app.js instead of inlining them in the html
- Update README to provide instructions

For issue #49

### CODE REVIEW

**Comment by edisonywh** [comment]
There's quite a few functions that you can delete at `lib/backoffice/layout_view.ex`

**Comment by NduatiK** [comment]
Oh wow. I hadn't seen these. I'll get rid of them.

File: lib/backoffice/layout.ex [hunk 1]
```diff
@@ -5,4 +5,5 @@ defmodule Backoffice.Layout do
   @callback logo() :: String.t()
   @callback stylesheets() :: [String.t()]
   @callback scripts() :: [String.t()]
+  @callback static_path() :: String.t()
```
Comment (edisonywh) [inline lib/backoffice/layout.ex:8]: I think this should be optional callback, in fact, I think all callbacks in here are optional, can you make that update?

Comment (edisonywh) [reply lib/backoffice/layout.ex:8]: We could also do that in a different PR actually

**Review by edisonywh — COMMENTED** [review]
I tried to run it with the [backoffice demo](https://github.com/edisonywh/backoffice-demo) app but I'm getting:

```elixir
[debug] ** (Phoenix.Router.NoRouteError) no route found for GET /backoffice/js/app.js (MoonWeb.Router)
```

My brain isn't really functioning now, but if I had to wager a guess, I think it's happening because right now the static path is referring to the user app's priv/static path, but Backoffice's js & css files are in Backoffice's priv/static file path, so we need some way to get the path to Backoffice's priv/static path?

File: lib/backoffice/layout.ex (hunk 1 above)
Comment (NduatiK) [reply lib/backoffice/layout.ex:8]: I added instructions for referencing backoffice in the first commit [here](https://github.com/edisonywh/backoffice/pull/52/commits/fa41adc0b535fad2dddf1d5774aa714e0aef568c). You are correct, you will need to provide the following plug in your endpoint.ex
```
plug Plug.Static,    
  at: "/backoffice",    
  from: :backoffice,    
  gzip: false,    
  only: ~w(css js)
```

Comment (edisonywh) [reply lib/backoffice/layout.ex:8]: You're absolutely right. Great work!

**Pull request merged**